bundles/org.openhab.binding.miio/src/main/resources/database/nwt.derh.wdh318efw1.json

```diff
@@ -0,0 +1,189 @@
+{
+	"deviceMapping": {
+		"id": [
+			"nwt.derh.wdh318efw1"
+		],
+		"channels": [
+			{
+				"property": "on_off",
+				"friendlyName": "Power",
+				"channel": "power",
+				"type": "Switch",
+				"refresh": true,
+				"ChannelGroup": "Power",
+				"actions": [
+					{
+						"command": "set_power",
+						"parameterType": "ONOFF"
+					}
+				],
+				"category": "switch",
+				"tags": [
+					"Switch"
+				]
+			},
+			{
+				"property": "mode",
+				"friendlyName": "Mode",
+				"channel": "mode",
+				"type": "String",
+				"refresh": true,
+				"ChannelGroup": "actions",
+				"actions": [
+					{
+						"command": "set_mode",
+						"parameterType": "STRING"
+					}
+				],
+				"tags": [
+					"Control"
+				]
+			},
+			{
+				"property": "auto",
+				"friendlyName": "Auto humidity",
+				"channel": "autohumidity",
+				"type": "Number",
```

Comment:
Should this be a percentage?

Reply:
@feodor85 can you comment on that

Reply:
My file is completely working. If you have any comments, please send a new file and I'll test it.

Reply:
I think the question of @cpmeister is if this is a percentage in the app, as then the number is better presented as a quantityType Number:Dimensionless with unit: percentage.

It will work both ways, but the latter is more in line of OH3 semantic model line of thinking

Reply:
do i need to change to and check?

```
"type": "Number:Dimensionless",
"unit": "PERCENT",
```

Reply:
I think that is all that would need to change, but @marcelrv would have to confirm that.

Reply:
indeed 👍